core/sdk-core/src/test/java/software/amazon/awssdk/core/util/DefaultSdkAutoConstructListTest.java

```diff
@@ -0,0 +1,29 @@
+package software.amazon.awssdk.core.util;
```

Comment:
The test files are missing copyright header. You can copy paste the copyright header from other file in this repo